app/views/layouts/_footer.html.erb

```diff
@@ -10,6 +10,7 @@
       </ul>
     </section>
     <ul class="copyright">
+      <li> <%= link_to "Become Host Shelter", host_shelters_sign_up_path %> </li>
```

Comment:
Where Should this really go?? I stuck it here for testing purposes and to keep layout clean, and it can stay there, but it will be hard to find for Users